pytorch_tabular/config/config.py

```diff
@@ -598,38 +624,23 @@ def __post_init__(self):
                 if self.metrics_params is None
                 else self.metrics_params
             )
+        elif self.task == "ssl":
+            assert self.ssl_task, "if task is ssl, ssl_task cannot be None"
+            assert self.aug_task, "if task is ssl, aug_task cannot be None"
+            if self.ssl_task == "Contrastive":
+                self.loss = "ContrastiveLoss" if self.loss is None else self.loss
+            else:
+                self.loss = "MSELoss" if self.loss is None else self.loss
+            self.metrics = (
```

Comment:
What happens with metrics for SSL? Should we restrict it to regression metrics?

Reply:
In case of SSL I think that we can use the regression metrics, since it's all about a reconstruction loss

Reply:
This ties back to the discussion we have on LinkedIn.. So maybe we can take this separately..